### DISCUSSION TITLE
Support scrolling on the flow editor when a older revision is selected.

### DISCUSSION THREAD

**Discussion by norkans7** [nyaruka/floweditor#1197] [Ideas]
When trying to view the past revision of a flow without reverting to it, one cannot scroll to get a hint of what nodes were not yet added.

Having a way to interact with the past revision would be helpful before reverting to it if needed or just for customer support to understand what could have been the flow used by a flow run.

**Comment by ericnewcomer** [comment]
Totally agree. Need to rework immutability. Interaction is currently disabled because saving throughout the editor is implicit so to avoid saving a revision (effectively reverting) we block interaction.
